sdks/java/harness/src/test/java/org/apache/beam/fn/harness/BoundedSourceRunnerTest.java

```diff
@@ -146,6 +146,7 @@ public void testCreatingAndProcessingSourceFromFactory() throws Exception {
         Suppliers.ofInstance("57L")::get,
         ImmutableMap.of(),
         ImmutableMap.of(),
+        ImmutableMap.of(),
```

Comment:
<!--new_thread; commit:5300be61ccac7a006ec7f2b8f3824cddb4dc75e8; resolved:1-->
Nit: `Collections.emptyMap()` is even more clear, despite the surrounding code.

Reply:
Changed.